public/Lab-Files/Lesson02/Challenges/text-level-elements.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+<p>Web development is a broad term for the work involved in developing a web site for the Internet <em>(World Wide Web)</em> or an intranet <em>(a private network)</em>. Web development can range from developing the simplest static single page of plain text to the most complex web-based internet applications, electronic businesses, and social network services. A more comprehensive list of tasks to which web development commonly refers, may include web engineering, web design, web content development, client liaison, client-side/server-side scripting, web server and network security configuration, and e-commerce development. Among web professionals, <strong>"web development"</strong>usually refers to the main non-design aspects of building web sites: writing markup and coding. Most recently Web development has come to mean the creation of content management systems or <strong>CMS</strong>. These <strong>CMS</strong> can be made from scratch, proprietary or open source. In broad terms the <strong>CMS</strong> acts as middleware between the database and the user through the browser. A principle benefit of a <strong>CMS</strong> is that it allows non-technical people to make changes to their web site without having technical knowledge.+
+
+</p>
+
+    
+</body>
+</html>
```

Comment:
Great job! Again, there are no errors on your code, but the spacing is a bit off.
You do not need 2 blank lines (11-12), or the blank line on line 9.

Your code is great though, I can tell you have been validating. Keep up the good work.